README.md

```diff
@@ -25,30 +25,58 @@
 
 # cordova-fetch
 
-This module is used for fetching modules from npm and gitURLs. It fetches the modules via `npm install`. It can also `npm uninstall` modules from a project.
+This package can be used to install and uninstall Node.js packages using npm.
 
-## Usage:
+## Usage
 
-### Fetching:
-```
-var fetch = require('cordova-fetch');
+### fetch
 
-fetch(spec, dest, opts);
-```
+Installs a module from npm, a git url or the local file system. Returns a `Promise` resolving to the absolute path to the installed package.
 
-`spec` can be a string containg a npm `packageID` or a `git URL`. 
-`dest` is string of the directory location you wish to `npm install` these modules.
-`opts` is an Object of options cordova fetch handles. Currently, fetch only support the `save` option.
-    eg. `{'save':true}`
+```js
+const fetch = require('cordova-fetch');
 
-### Removing:
+fetch(spec, dest, opts).then(pathToInstalledPackage => {
+    // Do something
+});
 ```
-var npmUninstall = require('cordova-fetch').uninstall;
 
-npmUninstall(spec, dest, opts);
+#### Parameters
+
+Parameter | Description
+-|-
+`spec` | A spec for the package to be installed (anything supported by `npm install`)
+`dest` | Location where to install the package
+`opts` | Additional options (optional)
+
+##### Options
+
+Option | Default | Description
+-|-
+`save` | `false` | Adds the package as  dependency to `package.json` iff `true`
```

Comment:
double space and "iff"

Reply:
[Iff](https://en.wikipedia.org/wiki/If_and_only_if) is not a typo. But given the fact that this mistake has been pointed out to me a few times before, I start to wonder whether I should continue to use it. :thinking: 

Would *If and only if* have been less irritating?

Reply:
It would consist of words that I know - but that's my problem ;) TIL...